src/v/redpanda/admin_server.cc

```diff
@@ -1689,6 +1694,11 @@ void admin_server::register_broker_routes() {
       ss::httpd::broker_json::stop_broker_maintenance,
       [this](std::unique_ptr<ss::httpd::request> req)
         -> ss::future<ss::json::json_return_type> {
+          if (!_controller->get_feature_table().local().is_active(
```

Comment:
I am wondering if we should add feature check to `members_frontend` ? This way using maintenance mode would be controlled centrally not only at API level.

Reply:
yes. I'll do both. the one in members_frontend as a central operation, but at the http layer as well so we can provide better error messages than stuff like "operation not supported" or whatever error a missing rpc would cause.